src/PhpPresentation/Shape/Chart/Type/Area.php

```diff
@@ -39,4 +39,9 @@ public function getHashCode(): string
 
         return md5($hash . __CLASS__);
     }
+
+    public function getXmlElementName(): string
```

Comment:
Could you remove `getXmlElementName` method for all chart types ? It's not compatilble with `Open Documentation Format` for `LibreOffice`.

Reply:
Sorry, I reverted my change in the chart types and added the xml element names to the data provider.

Let me know if you need more.